src/view/filter-view.js

```diff
@@ -1,16 +1,49 @@
 import { AbstractView } from 'frameworkView';
+import { FilterType } from 'const';
 
-const createFilterTemplate = () => (
-  `<nav class="main-navigation">
-    <a href="#all" class="main-navigation__item main-navigation__item--active">All movies</a>
-    <a href="#watchlist" class="main-navigation__item">Watchlist <span class="main-navigation__item-count">13</span></a>
-    <a href="#history" class="main-navigation__item">History <span class="main-navigation__item-count">4</span></a>
-    <a href="#favorites" class="main-navigation__item">Favorites <span class="main-navigation__item-count">8</span></a>
-  </nav>`
-);
+const ACTIVE_CLASS = 'main-navigation__item--active';
+
+const filtersNamesMap = {
+  [FilterType.ALL]: 'All movies',
+  [FilterType.WATCHLIST]: 'Watchlist',
+  [FilterType.HISTORY]: 'History',
+  [FilterType.FAVORITES]: 'Favorites',
+};
+
+const createFilterItemTemplate = (filter, isAllMovies) => {
```

Comment:
сложное название. 

```suggestion
const createFilterTemplate = (filter, isAllMovies) => {
```
Зачем указывать Item?